viewer/packages/camera-manager/src/ComboControls.ts

```diff
@@ -274,7 +286,7 @@ export default class ComboControls extends EventDispatcher {
     // @ts-ignore event.wheelDelta is only part of WebKit / Opera / Explorer 9
     if (event.wheelDelta) {
       // @ts-ignore event.wheelDelta is only part of WebKit / Opera / Explorer 9
-      delta = -event.wheelDelta / 40;
+      delta = -event.wheelDelta / 40; // Why division by 40?
```

Comment:
I don't know, why? Either add a TODO with an actionable, or remove the comment all together. This comment does not add anything.